Source/WebCore/loader/archive/cf/LegacyWebArchive.cpp

```diff
@@ -709,7 +709,7 @@ RefPtr<LegacyWebArchive> LegacyWebArchive::create(const String& markupString, Lo
                     continue;
 
                 auto addResult = uniqueSubresources.add(subresourceURL.string(), emptyString());
-                RefPtr<ArchiveResource> resource = documentLoader.subresource(subresourceURL);
+                RefPtr<ArchiveResource> resource = documentLoader->subresource(subresourceURL);
```

Comment:
Would be nice to drop `<ArchiveResource>` since you're modifying this line.

Reply:
Makes sense, done.